src/app/features/create-event/create-event.module.ts

```diff
@@ -51,6 +53,6 @@ import { detailsGuard } from './guards/details.guard';
       },
     ]),
   ],
-  providers: [CreateEventService, { provide: MAT_DATE_LOCALE, useValue: 'en-GB' }],
+  providers: [CreateEventFormService, { provide: MAT_DATE_LOCALE, useValue: 'en-GB' }],
```

Comment:
czy na pewno chcesz ten serwis rejestrować w module? czy nie chcemy być za każdym razem jhak użytkownik będzie chciał stworzyć event to właśnie tworzyć serwis, ktory będzie trzymał nam stan formularza?